client/views/add/add.js

```diff
@@ -87,27 +89,35 @@ Template.add.events({
         mods.push(modsInput[m].value);
       }
     }
-    Meteor.call('addMods', mods, template.data._id, (error, result) => {
-      // If the result is an object, there was an error
-      if (typeof result === 'object' && result.length > 0) {
+    Meteor.call('addMods', mods, template.data._id, (error, response) => {
+      console.log(response);
```

Comment:
@singhsanket143 can you remove it?

Reply:
@peter-hank Updated this log statements.